website/docs/developer-guide/chaoscenter-developer-guide.md

```diff
@@ -0,0 +1,219 @@
+notepad C:\Windows\System32\drivers\etc\hosts
+
+# add the below line
+127.0.0.1       m1 m2 m3
+```
+
+Linux/Mac
+```bash
+# add hosts in hosts 
+sudo vim /etc/hosts
+
+# add the below line
+127.0.0.1       m1 m2 m3
+```
+
+
+Step-3: Configure the mongoDB replica set
+
+```bash
+docker exec -it m1 mongo -port 27015
+
+config={"_id":"rs0","members":[{"_id":0,"host":"m1:27015"},{"_id":1,"host":"m2:27016"},{"_id":2,"host":"m3:27017"}]}
+
+rs.initiate(config)
+
+db.getSiblingDB("admin").createUser({user:"admin",pwd:"1234",roles:[{role:"root",db:"admin"}]});
+```
+
+### 2. Run the Authentication Server
+
+:::note
+Make sure to run backend services before the frontend. If you haven’t already cloned the litmus project do so from the `litmuschaos/litmus` repository
+:::
+
+```bash
+git clone https://github.com/litmuschaos/litmus.git litmus --depth 1
+```
+
+
+Step-1: Export the following environment variables
+
+```bash
+export ADMIN_USERNAME=admin
+export ADMIN_PASSWORD=litmus
+export DB_SERVER="mongodb://m1:27015,m2:27016,m3:27017/?replicaSet=rs0"
+export DB_USER=admin
+export DB_PASSWORD=1234
+export REST_PORT=3000
+export GRPC_PORT=3030
+```
+
+Windows  
+Docker or Hyper-V is reserving that port range. You can use 3030 ports by running the command below
+
+```shell
+netsh interface ipv4 show excludedportrange protocol=tcp
+net stop winnat
+netsh int ipv4 add excludedportrange protocol=tcp startport=3030 numberofports=1
+net start winnat
+```
+
+Step-2: Run the go application
+
+```bash
+cd chaoscenter/authentication/api
+go run main.go
+```
+
+### 3. Run the GraphQL Server  
+
+Step-1: Export the following environment variables
+
+```bash
+export DB_SERVER="mongodb://m1:27015,m2:27016,m3:27017/?replicaSet=rs0"
+export DB_USER=admin
+export DB_PASSWORD=1234
+export JWT_SECRET=litmus-portal@123
+export PORTAL_ENDPOINT=http://localhost:8080
+export LITMUS_SVC_ENDPOINT=""
+export SELF_AGENT=false
+export INFRA_SCOPE=cluster
+export INFRA_NAMESPACE=litmus
+export LITMUS_PORTAL_NAMESPACE=litmus
+export PORTAL_SCOPE=namespace
+export SUBSCRIBER_IMAGE=litmuschaos/litmusportal-subscriber:ci
+export EVENT_TRACKER_IMAGE=litmuschaos/litmusportal-event-tracker:ci
+export CONTAINER_RUNTIME_EXECUTOR=k8sapi
+export ARGO_WORKFLOW_CONTROLLER_IMAGE=argoproj/workflow-controller:v2.11.0
+export ARGO_WORKFLOW_EXECUTOR_IMAGE=argoproj/argoexec:v2.11.0
+export CHAOS_CENTER_SCOPE=cluster
+export WORKFLOW_HELPER_IMAGE_VERSION=3.0.0
+export LITMUS_CHAOS_OPERATOR_IMAGE=litmuschaos/chaos-operator:3.0.0
+export LITMUS_CHAOS_RUNNER_IMAGE=litmuschaos/chaos-runner:3.0.0
+export LITMUS_CHAOS_EXPORTER_IMAGE=litmuschaos/chaos-exporter:3.0.0
+export ADMIN_USERNAME=admin
+export ADMIN_PASSWORD=litmus
+export VERSION=ci
+export HUB_BRANCH_NAME=v2.0.x
+export INFRA_DEPLOYMENTS="[\"app=chaos-exporter\", \"name=chaos-operator\", \"app=event-tracker\",\"app=workflow-controller\"]"
+export INFRA_COMPATIBLE_VERSIONS='["0.2.0", "0.1.0","ci"]'
+export DEFAULT_HUB_BRANCH_NAME=master
```

Comment:
We have updated the envs in the wiki, can you update them here as well?

Reply:
yes, sure. I'll update the wiki as well.